### DISCUSSION TITLE
matchImageSnapshot opens a new page to take the screenshots and that it is an issue

### DISCUSSION THREAD

**Discussion by melibe23** [cypress-io/cypress#16700] [Component Testing]
An element that I want to take a screenshot of hides when there is any other action on the site. So, even if I select correctly the element, the image appears empty

![image](https://user-images.githubusercontent.com/32499737/119800711-518a4c80-bedd-11eb-90a0-24d6a925e6cf.png)

![Compare basket -- Desktop -- dropdown_with_0_app snap](https://user-images.githubusercontent.com/32499737/119799384-4b47a080-bedc-11eb-911f-d655551136cf.png)

This is my config 
```
addMatchImageSnapshotCommand({
  failureThreshold: 0.09, // threshold for entire image
  failureThresholdType: 'percent', // percent of image or number of pixels
  customDiffConfig: { threshold: 0.01 }, // threshold for each pixel
  capture: 'viewport', // capture viewport in screenshot
})

```

This is my test:

```
describe(`Compare basket`, () => {
      context(`Desktop`, () => {
        before(() => {
          
          cy.visit(url)
          cy.get(appBar).find(`[data-testid="icon-ga-compare"]`).click({ force: true })
        })

        it(`dropdown_with_${index}_app`, () => {
          cy.get(`[data-testid="compare-basket"]`).matchImageSnapshot()
        })
      })
```

**Comment by jennifer-shehane** [comment]
Can you try to reproduce this with Cypress screenshot alone? That would determine whether it's an issue with Cypress or the match image snapshot plugin. 

```js
cy.get(`[data-testid="compare-basket"]`).screenshot()
```

**Reply by melibe23** [reply]
@jennifer-shehane Sorry that I didn't see your answer. Yes, I got the same behaviour. The issue, IMO, is:

the current behaviour of that element is `hide yourself when there is a scroll or the user clicks somewhere else` so screenshot command generate an unsolicited scroll and then the element is not longer present yet, is found.
